src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -42,6 +42,9 @@
     <Compile Include="System\Runtime\Serialization\SerializationException.cs" />
     <Compile Include="System\Runtime\Serialization\StreamingContext.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='' Or '$(TargetGroup)'=='netstandard1.7'">
```

Comment:
> I suppose he did it like this so that it continues to work in the future when the default is not netstandard1.7 anymore

I did.

> we should keep it as only checking if it is empty

Ok